qtranslate_hooks.php

```diff
@@ -225,6 +225,9 @@ function qtranxf_wpseo_replacements($replacements){
 add_filter('link_description', 'qtranxf_useCurrentLanguageIfNotFoundUseDefaultLanguage',0);
 add_filter('the_author', 'qtranxf_useCurrentLanguageIfNotFoundUseDefaultLanguage',0);
 
+add_filter('get_terms', 'qtranxf_useCurrentLanguageIfNotFoundUseDefaultLanguage',0);
+add_filter('term_description', 'qtranxf_useCurrentLanguageIfNotFoundUseDefaultLanguage',0);
```

Comment:
Well it's a bit of a cascade:
 
wp-includes\category-template.php:1230 calls:
`$description = get_term_field( 'description', $term, $taxonomy );`

which calls wp-includes\taxonomy.php:1503
`return sanitize_term_field($field, $term->$field, $term->term_id, $taxonomy, $context);`

which calls wp-includes\taxonomy.php:2276:
`apply_filters( "term_{$field}", $value, $term_id, $taxonomy, $context );`